### ISSUE TITLE
Conflicts with Kanban plugin dates and times

### ISSUE THREAD

**Issue by mgmeyers** [valentine195/obsidian-markdown-attributes#3]
**Describe the bug**
The Kanban plugin uses this format to define dates: `@{2021-09-14}` and this for times: `@@{12:00}`. This plugin attempts to parse these and throws an error:

```
DOMException: Failed to execute 'setAttribute' on 'Element': '2021-09-14' is not a valid attribute name.
    at HTMLParagraphElement.Element.setAttr (app://obsidian.md/app.js:1:125833)
    at MarkdownAttributes.eval (eval at <anonymous> (app://obsidian.md/app.js:1:1215379), <anonymous>:270:33)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app://obsidian.md/app.js:1:1215379), <anonymous>:31:71)
    at new Promise (<anonymous>)
    at __awaiter (eval at <anonymous> (app://obsidian.md/app.js:1:1215379), <anonymous>:27:12)
    at MarkdownAttributes.postprocessor (eval at <anonymous> (app://obsidian.md/app.js:1:1215379), <anonymous>:189:16)
    at Function.<anonymous> (app://obsidian.md/app.js:1:912634)
    at app://obsidian.md/app.js:1:120138
    at Object.next (app://obsidian.md/app.js:1:120243)
eval @ helpers.ts:126
```

Could this plugin ignore curly braces that don't contain a class or id definitions? Eg, you could use a regex like this to make sure they conform to your rules first: `/{\s*(?:(?:id=[^\s}]+?|\.[^\s}]+?)\s+)+}/`

**Markdown Source**
```
- [ ] This is pretty cool @{2021-09-12}
```

**Comment by valentine195** [comment]
I think it would be better on my end to wrap it in a try/catch block and fail silently if it’s not a valid attribute. The Pandoc version of this syntax allows *any* attribute to be set, such as `open` or `checked`.

**Comment by valentine195** [comment]
Hi, this conflict should be resolved in 1.0.2. Thank you for bringing it to my attention!

**Comment by mgmeyers** [comment]
Fantastic, thanks so much!
